src/connectionmanager.cpp

```diff
@@ -256,6 +256,12 @@ void ConnectionManager::quit() {
   }
 }
 
+void ConnectionManager::deleteTunnel() {
+  if (m_impl) {
+    m_impl->deleteTunnel();
+  }
+}
+
```

Comment:
Should we also check and attempt to delete the tunnel in `connectionManager::quit()`?

Reply:
Same comment as above.